travvy/src/components/AttractionsList.vue

```diff
@@ -25,12 +25,12 @@
       <!-- let the user go back to the home page and change the info they gave -->
       <router-link to="/Home" tag="button" class="button"><span>Change Preferences</span></router-link>
       <br><br>
-      <div class="btn-group">
-        <router-link to="/AttractionDetails" tag="button" class="button"><span>CN Tower</span></router-link>
-        <router-link to="/AttractionDetails" tag="button" class="button"><span>Ripleys</span></router-link>
-        <router-link to="/AttractionDetails" tag="button" class="button"><span>Royal Ontario Museum</span></router-link>
+      
+      <div class="btn-group" v-for="value in $store.state.recommendedAttractions" v-bind:key="value.id" >
+  
+          <button v-on:click="navigateTo({name:'AttractionDetails'})" tag="button" class="button"><span>{{ value.attraction_name }}</span></button>
```

Comment:
Great job on being able to figure out how to display the attraction names dynamically for each location, now we can use this same logic for the attraction details page